scripts/code_size_compare.py

```diff
@@ -146,7 +146,7 @@ def detect_arch() -> str:
         sys.exit(1)
 
 TFM_MEDIUM_CONFIG_H = 'configs/ext/tfm_mbedcrypto_config_profile_medium.h'
-TFM_MEDIUM_CRYPTO_CONFIG_H = 'configs/ext/crypto_config_profile_medium.h'
+TFM_MEDIUM_CRYPTO_CONFIG_H = 'tf-psa-crypto/configs/ext/crypto_config_profile_medium.h'
```

Comment:
I found that `all-core.sh` has a nice `TF_PSA_CRYPTO_ROOT_DIR` variable which seems to hold the path of `tf-psa-crypto`. Can't we use it here as we usually do with `$FRAMEWORK`?

Reply:
I guess we want this script to be usable outside of all.sh and more than that I think this script is not used in all.sh.

Reply:
Sorry, I picked the wrong file to add the comment. My intent was to add it in some of the components changed in `tests/scripts/` of course. So do you think that also in that case this change does not apply?
If so, I will just resolve the comment

Reply:
The scripts that are run by all.sh components use to work standalone and I think we should preserve that. Thus not use all.sh specific variables in them.